airgun/views/settings.py

```diff
@@ -16,6 +18,10 @@ class SettingsView(BaseLoggedInView, SearchableViewMixin):
         },
     )
 
+    @View.nested
+    class EmailView(View):
```

Comment:
You need to inherit from SatTab to have this auto-selecting magic to work:
```suggestion
    class EmailView(SatTab):
```

Import from `from airgun.views.common`.

Reply:
Fixed